maintainers/maintainer-list.nix

```diff
@@ -8420,4 +8420,10 @@
     github = "cpcloud";
     githubId = 417981;
   };
+  syd = {
+    name = "Tom Sydney Kerckhove";
+    email = "syd@cs-syd.eu";
+    github = "NorfairKing";
+    githubId = 3521180;
+  };
```

Comment:
The new entries should not go to the end of maintainer-list.nix. Please put them in their respective place according to the alphabetic ordering.